docs/contributor/04-10-technical-design.md

```diff
@@ -1,22 +1,22 @@
 # Technical Design
 
-## Kyma Istio operator
+## Kyma Istio Operator
 
-![Kyma IstioOperator Overview](./istio-operator-overview.svg)
+![Kyma IstioOperator Overview](./../assets/istio-operator-overview.svg)
```

Comment:
Diagrams in the doc don't follow these [guidelines on how to create diagrams](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/09-diagrams.md) in Kyma documentation.

Reply:
I've updated them